rules/Php80/Rector/NotIdentical/StrContainsRector.php

```diff
@@ -151,13 +151,4 @@ private function matchIdenticalOrNotIdenticalToFalse(Identical | NotIdentical |
 
         return null;
     }
-
-    private function isPositiveInteger(Expr $expr): bool
-    {
-        if (! $expr instanceof LNumber) {
-            return false;
-        }
-
-        return $expr->value > 0;
```

Comment:
How about direct 0 or negative int, could you add fixture for it to avoid regression?

Reply:
https://www.php.net/manual/en/function.substr.php

0 and negative values are totally valid offsets when considering strings though. I'm not sure why they would be excluded from this check.

Reply:
The fixture is needed to see before vs after with 0 or negative int to see the different

Reply:
Understood! Added. I changed it to remove substr() set if the offset is specified as zero (would be the start of the string).